client/js/components/user/CustomerSettings/CustomerSettingsSignLanguageVideo.vue

```diff
@@ -71,32 +69,33 @@
         v-model="video.description"
         reduced-height />
 
-      <dp-button
-        :busy="isBusy"
```

Comment:
why was the loading state removed here?

Reply:
because the other buttons in customersettings which have the same function has no busy and it has to be uniformly in all buttons for the parent component

Reply:
thanks, done here https://github.com/demos-europe/demosplan-core/pull/2212/commits/e09ca1f649d4ecf10ba9f280eba551c7c41900ed

Reply:
there were no loading states before because they were not needed because there was a form request submit which caused a reloading of the page. Now via api communication this is not the case anymore and clicking the buttons will not trigger a page reload, which made the loading states useful imho. thanks for adding them :)